File: Rregulloje.UI/RregullojeWebAPI/src/app/_ViewModels/ServiceTypesViewModel.ts
export class ServiceTypesViewModel {
  id: number;
  name: string;
  price: number;
  minServiceId: number;
  minService: string;

  constructor(id: number = null, name: string = null, price: number = null, minServiceId: number = null, minService: string = null) {
    this.id = id;
    this.name = name;
    this.price = price;
    this.minServiceId = minServiceId;
    this.minService = minService;
  }
}

File: Rregulloje.Application/ViewModels/ServiceTypesViewModel.cs
using Rregulloje.Domain.Models;
using System;
using System.Collections.Generic;
using System.Text;

namespace Rregulloje.Application.ViewModels
{
    public class ServiceTypesViewModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public double Price { get; set; }
        public int? MinServiceId { get; set; }
        public MinServices MinService { get; set; }
    }
}
